CHARTER.md

```diff
@@ -0,0 +1,73 @@
+
+
+Technical Charter (the “Charter”)
+for 
+Besu a Series of LF Projects, LLC
+
+Adopted ___________
+
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the Besu open source project, which has been established as Besu a Series of LF Projects, LLC (the “Project”).  LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, “Collaborators”) must comply with the terms of this Charter. 
+
+1.	Mission and Scope of the Project
+    a.	The mission of the Project is to develop an Ethereum client designed to be enterprise and infrastructure-friendly for both public and private permissioned network use cases, with an extractable EVM implementation.
+    b.	The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
+
+2.	Technical Steering Committee
+    a.	The Technical Steering Committee (the “TSC”) will be responsible for all technical oversight of the open source Project. 
+    b.	The TSC voting members are initially the Project’s Committers. At the inception of the project, the Committers of the Project will be as set forth within the “CONTRIBUTING” file within the Project’s code repository. The TSC may choose an alternative approach for determining the voting members of the TSC, and any such alternative approach will be documented in the CONTRIBUTING file.  Any meetings of the Technical Steering Committee are intended to be open to the public, and can be conducted electronically, via teleconference, or in person. 
+    c.	TSC projects generally will involve Contributors and Committers. The TSC may adopt or modify roles so long as the roles are documented in the CONTRIBUTING file. Unless otherwise documented: 
+        i.	Contributors include anyone in the technical community that contributes code, documentation, or other technical artifacts to the Project; 
+        ii.	Committers are Contributors who have earned the ability to modify (“commit”) source code, documentation or other technical artifacts in a project’s repository; and
```

Comment:
I am confused by the Comitter distinction here. I don't think we have defined this role anywhere. Only Maintainers.

Reply:
Think that's similar to what LF is using already?
https://docs.linuxfoundation.org/lfx/project-control-center/v2-latest-version/reports/health-metrics/code-contributions
I would guess a committer is what a maintainer is ATM but a maintainer for LF has increased responsibilities of releasing as well.
Anyway there could be at least a paragraph or a link explaining that though.